browser/brave_content_browser_client.cc

```diff
@@ -27,11 +27,13 @@
 #include "brave/components/brave_shields/browser/buildflags/buildflags.h"  // For STP
 #include "brave/components/brave_shields/browser/tracking_protection_service.h"
 #include "brave/components/brave_shields/common/brave_shield_constants.h"
+#include "brave/components/brave_wallet/browser/buildflags/buildflags.h"
 #include "brave/components/brave_webtorrent/browser/buildflags/buildflags.h"
 #include "brave/components/content_settings/core/browser/brave_cookie_settings.h"
 #include "brave/components/services/brave_content_browser_overlay_manifest.h"
 #include "brave/components/services/brave_content_packaged_service_overlay_manifest.h"
 #include "brave/grit/brave_generated_resources.h"
+#include "brave/browser/extensions/brave_wallet_navigation_throttle.h"
```

Comment:
shouldn't this also be below in BRAVE_WALLET_ENABLED + ENABLE_EXTENSIONS guard?

Reply:
fixed.